packages/eslint-config/index.js

```diff
@@ -319,13 +323,24 @@ module.exports = {
          * Use a more strict `assert` instead.
          */
         'no-restricted-imports': [...xo.rules['no-restricted-imports'], 'assert'],
+        'no-warning-comments': [1, { terms: ['todo', 'fixme'], location: 'start' }],
         // https://github.com/prettier/eslint-config-prettier#curly
         curly: ['error', 'all'],
         /**
-         * eslint-config-xo-typescript disabled this rule by default, need to enable it
+         * Eslint-config-xo-typescript disabled this rule by default, need to enable it
          * https://github.com/xojs/eslint-config-xo-typescript/blob/main/index.js#L446
          */
         '@typescript-eslint/no-non-null-assertion': 'error',
+        '@typescript-eslint/member-ordering': [
```

Comment:
which case are we intent to modify the default rule settings?

Reply:
logto/js/packages/client/src/index.ts
![image](https://user-images.githubusercontent.com/15182327/146333700-e9e1f5cf-b454-4cbf-8900-480f32349bbe.png)

This rule is added to ensure the order when we try to define a class and all its members (including constructor, fields and methods).